pydrive2/test/test_fs.py

```diff
@@ -116,8 +116,11 @@ def test_rm(fs, remote_dir):
     assert not fs.exists(remote_dir + "/dir/c/a")
 
 
-def test_ls(fs, remote_dir):
-    fs.mkdir(remote_dir + "dir/")
+def test_ls(fs: GDriveFileSystem, remote_dir):
+    _, base = fs.split_path(remote_dir + "dir/")
+    fs._path_to_item_ids(base, create=True)
+    assert fs.ls(remote_dir + "dir/") == []
```

Comment:
Turns out, `fs.mkdir()` is not implemented in GDriveFileSystem. I was not sure if it was needed as it has been working fine for so long. So I only fixed in test to create a directory.